### ISSUE TITLE
`Ctrl+w` cause copy replacing the first item in the clipboard

### ISSUE THREAD

**Issue by EricWvi** [warpdotdev/Warp#1398]
### Describe the bug

```text
`Ctrl+w` to delete one word, it will copy the word and replace the first item in the clipboard
```

### To Reproduce

Steps to reproduce:
1. copy one sha `f20577`
2. use up arrow key to get the history command `git show eb9f28`
3. `Ctrl+W`, and the buffer now is `git show `
4. paste, and it will be `git show eb9f28`, not `git show f20577`, because `Ctrl+W` cause a copy.

### Expected behaviour

```text
`Ctrl+W` won't trigger a copy.
```

### Operating System

MacOS

### OS Version

12.3.1

### Shell Version

zsh 5.8 (x86_64-apple-darwin21.0)

**Comment by elviskahoro** [comment]
Thanks for reporting will cirlce back when there are updats

**Comment by ls-ivan-kiselev** [comment]
That's literally the only reason I keep returning back to iTerm2

**Comment by EricWvi** [comment]
@ls-ivan-kiselev Now I am totally accustomed to this behavior. XD

We should not expect it to be a traditional terminal. Instead, we can use it in a warp-ish way. 

Basically, we can use it just as a modern gui text editor, say vscode. `Cmd+A`, `Cmd+<arrow>`, `Option+<arrow>`, moving cursor with mouse, selecting text with mouse, etc. All is supported.

As to deleting a word backward, we shoud use `Option+Backspace` instead. This will not conflict with your custom if you treat it as a gui text editor in your mind. And you will use `Ctrl+W` unconsciously when using shell in another situation, say IDE built-in emulated shell, since you treat it as a terminal.

I highly recommend people who have similar strange feelings giving it a try. I use Warp nearly the whole of 2022. Warp has many new features compared to when I first met it. I think it is really awesome.

**Comment by Lyt99** [comment]
I found that the behavior can be changed by remapping `Ctrl+W` to `Delete word left` instead of `Cut word left` in keyboard shortcuts. 

![image](https://user-images.githubusercontent.com/3164435/204688756-2d49a464-5c4c-4324-90a8-9b54542bbeba.png)

And I put `Option+Backspace` into `Cut word left`, but I rarely use it in my daily work :)

**Comment by EricWvi** [comment]
@Lyt99 yeah, it works.

**Comment by heng4fun** [comment]
@Lyt99 thx, you saved my day!

**Comment by akwodkiewicz** [comment]
This has been resolved in `v0.2023.04.25.08.05.stable_00`
<img width="536" alt="Screenshot 2023-04-28 at 16 29 57" src="https://user-images.githubusercontent.com/22861194/235176031-4d70627a-07dd-4be5-9573-8c284a4769f3.png">

**Comment by dannyneira** [comment]
Hey Folks! Just wanted to give you a quick update - as @akwodkiewicz mentioned, we've fixed the issue with the latest version of Warp `v0.2023.04.25.08.05.stable_00`.

I'll go ahead and mark this task as complete, but if you notice anything else, feel free to give us a heads-up. Thanks!
